### PR TITLE
Invalid api-conventions URL link

### PR DESCRIPTION
**Pull request by c3y1huang**
**What type of PR is this?**

/kind cleanup
/kind documentation

**What this PR does / why we need it**:
Original URL link is invalid.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: config/crd/snapshot.storage.k8s.io_volumesnapshotclasses.yaml [hunk 1]
```diff
@@ -27,7 +27,8 @@ spec:
         apiVersion:
           description: 'APIVersion defines the versioned schema of this representation
             of an object. Servers should convert recognized schemas to the latest
-            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#resources'
+            internal value, and may reject unrecognized values. 
+            More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
```
Comment (xing-yang) [inline config/crd/snapshot.storage.k8s.io_volumesnapshotclasses.yaml]: Those are generated automatically by running hack/update-crd.sh so we are not supposed to modify them manually.

Comment (c3y1huang) [reply config/crd/snapshot.storage.k8s.io_volumesnapshotclasses.yaml]: Reverted URL update for CRDs.

**Comment by xing-yang** [comment]
@yuxiangqian PTAL.

**Comment by xing-yang** [comment]
CI failed because you also have to update generated code after modifying types.go.

Take a look of the following issue.  Generating code is getting complicated after we changed the package path for v2:
https://github.com/kubernetes-csi/external-snapshotter/issues/239#issuecomment-577852778

**Comment by c3y1huang** [comment]
> CI failed because you also have to update generated code after modifying types.go.
> 
> Take a look of the following issue. Generating code is getting complicated after we changed the package path for v2:
> [#239 (comment)](https://github.com/kubernetes-csi/external-snapshotter/issues/239#issuecomment-577852778)

Thanks, CI failed due to vendor changes. Didn't know what I was thinking :-/. Revert to address. 
PTAL.

**Comment by xing-yang** [comment]
Just noticed that this other PR (https://github.com/kubernetes-csi/external-snapshotter/pull/256) which was submitted a few days earlier than this PR already contains changes in this PR.  Let's put this PR on hold for now, wait until the other PR is merged, and then see if there're any additional things that need to be addressed.

**Comment by humblec** [comment]
@c3y1huang the PR changs intended already got in via #260 , unless if some other changes are planned with this PR, I believe we could close this one.. Thanks! @xing-yang what do you think ?

**Pull request closed without merging**